lib/senkyoshi.rb

```diff
@@ -103,7 +111,7 @@ def self.initialize_course(canvas_file_path, blackboard_file_path)
     end
   end
 
-  def true?(obj)
+  def self.true?(obj)
```

Comment:
I made this change because I was wanted to use this method in a class method. I couldn't figure out a way to use a module instance method in a class method. 

I also noticed that this was the only non static method in this file. Is there something about this method in particular that makes it more suited to be an instance method?